recipes-kernel/linux/meson64-kmeta/bsp/khadas-vim3-standard.scc

```diff
@@ -0,0 +1,7 @@
+define KMACHINE khadas-vim3
```

Comment:
Cant we have a single scc for all 64bit boards ? essentially the goal is to have a single kernel setup for all 64bit amlogic boards. The VIM3 has nothing specific to be enabled the other 64bits boards need.
Would it take the `meson-gx` name ? or it must be the machine name ?

Reply:
I believe it needs to be the machine name, but let me explore.  I'll try and get something back this evening.

Reply:
As for a single scc for all boards, yes we could, but I was following the example of meta-arm Juno definition which has the features as separate.  This way, if I wanted to remove the function from the built kernel, it is obvious which I should remove.
My personal feeling is that it is cleaner to have the functions separated out.  My personal preference would be to keep it this way.

Reply:
Yep but due to the huge number of boards, it's hardly maintainable.
But if we can have a `meson-gx.scc` and some board-specific scc, it would be the best so we can have a generic kernel and some custom settings for some boards

Reply:
Ok, so I have merged the config into a single .cfg file now.